README.md

```diff
@@ -0,0 +1,12 @@
+# GoThumb
+
+A [golang](http://golang.org/) port of [thumbor](https://github.com/thumbor/thumbor)
+
+## Features
+
+- [x] Image Resizing via URL
+- [x] HMAC url signing
+- [x] Caching of resized images in S3
+- [ ] Smart crop support
+- [ ] Concurrent S3 bucket downloads
```

Comment:
actually it does parallel downloads on already-cached images, and parallel uploads when it stores those images. It just doesn't fetch source files in parallel yet